src/screens/TopicMemberScreen/index.js

```diff
@@ -207,4 +208,4 @@ const TopicMemberScreen = () => {
     </SafeAreaProvider>
   );
 };
-export default TopicMemberScreen;
+export default withInteractionsManaged(TopicMemberScreen);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Wrapping the exported component with `withInteractionsManaged` is a good practice for managing interactions in React Native, especially when dealing with animations or other intensive tasks. Just ensure that `withInteractionsManaged` is implemented correctly and doesn't interfere with any existing functionality.
```diff
- export default TopicMemberScreen;
+ export default withInteractionsManaged(TopicMemberScreen);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->